pkg/network/ovn_kubernetes.go

```diff
@@ -1455,11 +1451,25 @@ func shouldUpdateOVNKonPrepull(ovn bootstrap.OVNBootstrapResult, releaseVersion
 	return true, false
 }
 
-// isUserIPsecMachineConfigPresent returns true if user owned MachineConfigs for IPsec plugin
-// are already present either in master or worker nodes, otherwise returns false.
-func isUserIPsecMachineConfigPresent(infra bootstrap.InfraStatus) bool {
-	return (infra.MasterIPsecMachineConfig != nil && !containsNetworkOwnerRef(infra.MasterIPsecMachineConfig.OwnerReferences)) ||
-		(infra.WorkerIPsecMachineConfig != nil && !containsNetworkOwnerRef(infra.WorkerIPsecMachineConfig.OwnerReferences))
+func isCNOIPsecMachineConfigExists(infra bootstrap.InfraStatus) bool {
```

Comment:
nit: `isCNOIPsecMachineConfigPresent`?

The doc text is always useful, why not keep it around since it was already there?

Reply:
done.